Mint.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Mint Page Project Microverse</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+  <link rel="stylesheet" href="Mint.css" />
+</head>
+
+<body>
+  <header>
+    <button type="submit"><i class="fa fa-lock"></i> Sign In</button>
+    <p>
+        Already have an account?<br>
+      <a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS#"
+       >I forgot my User ID or Password</a>
+    </p>
+  
+  
+  </header>
+  <section>
```

Comment:
- [ ] **[OPTIONAL]** For correct HTML semantics, try to use the `<main>` tag to encapsulate the main part of your page, in this case, instead of the  `<section>` tag

Reply:
ok thanks that's very  helpful it's corrected now